maintainers/maintainer-list.nix

```diff
@@ -7187,6 +7187,12 @@
     githubId = 165283;
     name = "Alexey Kutepov";
   };
+  rgrunbla = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add rgrunbla``.